manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -373,7 +373,14 @@ data:
       {{- if .Values.global.proxy.lifecycle }}
         lifecycle:
           {{ toYaml .Values.global.proxy.lifecycle | indent 4 }}
-        {{- end }}
+      {{- else if .ProxyConfig.HoldApplicationUntilProxyStarts}}
+        lifecycle:
```

Comment:
What if `.Values.global.proxy.lifecycle ` is non-empty? Are multiple lifecycle directives supported?